.github/workflows/tii-mesh-com.yaml

```diff
@@ -0,0 +1,47 @@
+name: tii-mesh-com
+
+on:
+  push:
+    branches: main
+    tags:
+      - 'v*'
+      - v[0-9]+.[0-9]+.[0-9]+
+      - v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+
+  pull_request:
+    branches: [ main, develop ]
+
+jobs:
+  tii-mesh-com:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - uses: docker/setup-buildx-action@v1
+
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/tiiuae/tii-mesh-com
+          tags: |
+            type=ref,event=branch
+            type=semver,pattern={{version}}
+            type=sha
+            type=raw,value=latest
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        if: github.event_name == 'push'
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build tii-mesh-com image and push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./modules/mesh_com/Dockerfile
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
please also add:

```
labels: ${{ steps.meta.outputs.labels }}
```

Labels carry OCI-compliant container metadata, along with them is Git revision. The Git revision is required for `$ fog manifest update-component` tool to update both the container image reference and Git revision reference at the same time.

Reply:
Added labels